zeebe/util/src/main/java/io/camunda/zeebe/util/HashUtil.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.util;
+
+import com.google.common.hash.Hashing;
+import java.nio.charset.StandardCharsets;
+
+/** Utility class for generating hash values. */
+public final class HashUtil {
+
+  private HashUtil() {}
+
+  /**
+   * Computes a 64-bit hash value from the given string using Murmur3 128-bit hashing.
+   *
+   * @param stringValue the input string to hash
+   * @return the 64-bit hash value of the input string
+   */
+  public static long getStringHashValue(final String stringValue) {
+    return Hashing.murmur3_128().hashString(stringValue, StandardCharsets.UTF_8).asLong();
```

Comment:
💭 I'd still recommend to document why we picked this hash algorithm over the others. Java also offers a couple of built in algorithms that wouldn't even require a dependency.
It could also be a follow-up to minimize conflicts.
This is a good use for AI as well to discover and compare options.